src/strings.c

```diff
@@ -1824,3 +1824,6 @@ const u8 gText_Answer[] = _("ANSWER");
 const u8 gText_PokeBalls[] = _("POKé BALLS");
 const u8 gText_Berry[] = _("BERRY");
 const u8 gText_Berries[] = _("BERRIES");
+const u8 gText_ExpShareOn[] = _("Turned on the Exp. Share.\pParty will now gain a portion\nof the Experience Points.{PAUSE_UNTIL_PRESS}");
+const u8 gText_ExpShareOff[] = _("Turned off the Exp. Share.\pParty will no longer gain a portion\nof any Experience Points.{PAUSE_UNTIL_PRESS}");
+const u8 gText_FeatureUnavailableConfigItem[] = _("Feature unavailable!\nPlease define a usable flag in:\l'include/config/item.h'!{PAUSE_UNTIL_PRESS}");
```

Comment:
This is a really good idea!

However, would it be too crazy to instead use a `STATIC_ASSERT` to force you to have a non-zero flag if you have gen 6+ behavior? I generally prefer throwing a compile-time error when the user has done something that's a mistake, rather than waiting until run-time to detect it (and especially something like this where if they never think to test the exp share, it simply won't work).

Reply:
Hmm, I suppose. Then the default for this setting would then be `GEN_5`.

Is there a way to send a custom error message to let them know how they should use it, instead of them coming to the server and ask why it isn't working?

Reply:
Sadly it's tricky to get a great error message :(

GriffinR's approach is to write a comment on the line above, and to try and give the `id` parameter a name that's somewhat evocative of what's gone wrong. e.g. we might do `STATIC_ASSERT(..., Gen6ExpShareNeedsFlag);` or something.

Reply:
Alright, will try to implement it.

Maybe we could do something similar to these later:
![image](https://user-images.githubusercontent.com/2904965/232170038-1462e67e-05a2-4551-971c-58513feda845.png)